image_client.py

```diff
@@ -116,7 +117,14 @@ def upload_to_image_server(self, full_path, redacted, collection, original_path=
         url = self.build_url("fileupload")
         logging.debug(f"Attempting upload to {url}")
         r = requests.post(url, files=files, data=data)
```

Comment:
fix to prevent undatabased imports from overwriting the html of databased imports, already in prod as a duct tape fix to get imports running, but adding to commit to make it official.